docs/tutorials/embassy/index.md

```diff
@@ -66,7 +66,10 @@ Options:
   -s, --serial   Connect to serial after deploy
   -h, --help     Print help
 ```
-
+:::info Before installing elf2uf2-rs, you might need an additional package, called libudev. You can get it by running the following in your terminal. :::
```

Comment:
```suggestion
:::info Before installing elf2uf2-rs, you might need an additional package, called `libudev`. You can get it by running the following in your terminal. :::
```

Reply:
Also added special formatting to `libudev`